src/delimiter.js

```diff
@@ -331,74 +373,44 @@ const makeStackedDelim = function(
-    // 0.005em above the lower end of the top element.
-    const shiftOfExtraElement = (repeatCount + 1) * 0.005 - repeatHeightTotal;
 
     // Now, we start building the pieces that will go into the vlist
-
-    // Keep a list of the inner pieces
-    const inners = [];
+    // Keep a list of the pieces of the stacked delimiter
+    const stack = [];
 
     // Add the bottom symbol
-    inners.push(makeInner(bottom, font, mode));
+    stack.push(makeCorner(bottom, font, mode));
+    stack.push(lap); // overlap
 
     if (middle === null) {
-        // Add that many symbols
-        for (let i = 0; i < repeatCount; i++) {
-            inners.push(lap); // overlap
-            inners.push(makeInner(repeat, font, mode));
-        }
+        // The middle section will be an SVG. Make it an extra 0.016em tall.
+        // We'll overlap by 0.008em at top and bottom.
+        const innerHeight = realHeightTotal - topHeightTotal - bottomHeightTotal
+            + 0.016;
+        stack.push(makeInner(repeat, innerHeight, options));
     } else {
         // When there is a middle bit, we need the middle part and two repeated
         // sections
-        for (let i = 0; i < repeatCount; i++) {
-            inners.push(lap);
-            inners.push(makeInner(repeat, font, mode));
-        }
-        // Insert one extra repeat element.
-        inners.push({type: "kern", size: shiftOfExtraElement});
-        inners.push(makeInner(repeat, font, mode));
-        inners.push(lap);
+        const innerHeight = (realHeightTotal - topHeightTotal - bottomHeightTotal -
+            middleHeightTotal) / 2 + 0.016;
+        stack.push(makeInner(repeat, innerHeight, options));
         // Now insert the middle of the brace.
-        inners.push(makeInner(middle, font, mode));
-        for (let i = 0; i < repeatCount; i++) {
-            inners.push(lap);
-            inners.push(makeInner(repeat, font, mode));
-        }
-    }
-
-    // To cover the gap create by the overlaps, insert one more repeat element,
-    // at a position that juts 0.005 above the bottom of the top element.
-    if ((repeat === "\u239c" || repeat === "\u239f") && repeatCount === 0) {
-        // Parentheses need a short repeat element in order to avoid an overrun.
-        // We'll make a 0.3em tall element from a SVG.
-        const overlap = buildCommon.svgData.leftParenInner[2] / 2;
-        inners.push({type: "kern", size: -overlap});
-        const pathName = repeat === "\u239c" ? "leftParenInner" : "rightParenInner";
-        const innerSpan = buildCommon.staticSvg(pathName, options);
-        inners.push({type: "elem", elem: innerSpan});
-        inners.push({type: "kern", size: -overlap});
-    } else {
-        inners.push({type: "kern", size: shiftOfExtraElement});
-        inners.push(makeInner(repeat, font, mode));
-        inners.push(lap);
+        stack.push(lap);
+        stack.push(makeCorner(middle, font, mode));
```

Comment:
It feels a little weird call `makeCorner` here but I don't have a suggestion for a better name for the function.  It might be worth add a short comment here saying the `makeCorner` is used to create `VListElem`s for all the non-stretchy parts of delimiters.

Reply:
Yep, I had the same thought as I was naming the variable.

Reply:
I ended up renaming that function to `makeGlyphSpan`.